### ISSUE TITLE
1.5 model has problems with patterns, that model 1.2 didn't have.

### ISSUE THREAD

**Issue by hzwer** [hzwer/arXiv2020-RIFE#65]
I received a mail that contained some bad cases.
v1.5
<img width="300" alt="image" src="https://user-images.githubusercontent.com/10103856/101895689-d3621400-3be2-11eb-94d9-dde24c7c2750.png"> <img width="450" alt="image" src="https://user-images.githubusercontent.com/10103856/101895710-db21b880-3be2-11eb-88ea-7622f7e8a511.png">

**Comment by hzwer** [comment]
WIP v1.6
![0000000](https://user-images.githubusercontent.com/10103856/101896714-58016200-3be4-11eb-9335-27ac772c480b.png)

**Comment by StoneCypher** [comment]
1.6 has a lesser defect in the top row of the building just below and to our left of the chimney in the center

what is the model difference?  just other images in the training set or something?

**Comment by hzwer** [comment]
We are adjusting the augmentation on our training set. Only some fine-tuning of the model is currently possible, because I am working on a new frame interpolation algorithm.

**Comment by ChristianUndCo** [comment]
OK - Model 1.6 is better with patterns than 1.5 but still have big problems with it.
Model 1.2 was much better at patterns but had other issues.

**Comment by ChristianUndCo** [comment]
If you need content to test, I can deliver....

**Comment by hzwer** [comment]
v1.7
<img width="300" alt="image" src="https://user-images.githubusercontent.com/10103856/103265564-dd5c8300-49e8-11eb-817f-62c961397dfa.png">
<img width="450" alt="image" src="https://user-images.githubusercontent.com/10103856/103265607-ef3e2600-49e8-11eb-9dda-89986e4fe3af.png">

**Comment by StoneCypher** [comment]
lol i was super grossed out by this defect on the building until i realized that it was a smudge on my screen

this is nice looking stuff

**Comment by ChristianUndCo** [comment]
New models still have problems with patterns. The differences between the results are very small from model to model. I made a collection of clips that cause problems. Here you can download the sourceclips (25fps) and try yourself. 

https://www.magentacloud.de/share/hn-zeqn0dj

![Fensterladen](https://user-images.githubusercontent.com/76535067/103527074-19c52e80-4e82-11eb-9a28-b4743a6d9fa8.jpg)
![Hochhaus](https://user-images.githubusercontent.com/76535067/103527076-1a5dc500-4e82-11eb-8b37-990103b905b3.jpg)
![Jalousie](https://user-images.githubusercontent.com/76535067/103527077-1af65b80-4e82-11eb-99fd-35e1f7183787.jpg)
![MusteramArm](https://user-images.githubusercontent.com/76535067/103527155-37929380-4e82-11eb-89a9-a199194514e1.jpg)
![Torzaun](https://user-images.githubusercontent.com/76535067/103527156-382b2a00-4e82-11eb-90c1-dfa0f9a16b51.jpg)
![Vogelkaefig](https://user-images.githubusercontent.com/76535067/103527158-382b2a00-4e82-11eb-9e4e-a3939dac706c.jpg)

**Comment by ChristianUndCo** [comment]
Model 2.0 is quite good, but still has problems with patterns.
Try my sourceclips 
https://www.magentacloud.de/share/hbx2la7qrk

**Comment by Tsubajashi** [comment]
Have you checked model 2.2? havent tested it myself with the source clips, but my pattern problems in my clips i tested are almost completely gone

**Comment by ChristianUndCo** [comment]
Yes, I check each new model with my sourceclips. In some cases the result is better but not completely gone. Still enough problems to bother the filmfun.

**Comment by hzwer** [comment]
v2.3 is uploaded. Such problems have been greatly alleviated.

**Comment by hzwer** [comment]
![image](https://user-images.githubusercontent.com/10103856/109099898-d474fd00-775e-11eb-8095-97f41f91f9af.png)

**Comment by ChristianUndCo** [comment]
Model 2.3 brought a very very (very) small improvement of this stuff. But there is nearly no visible change to model 2.2. These clips cause still big problems for Rife :(

**Comment by hzwer** [comment]
@ChristianUndCo I add a flag, you can set scale=2.0 or scale=4.0 to get better result for your cases.

**Comment by Fadexz** [comment]
Not sure if you would like to take a look at these but these are just some videos I had issues with bad artifacting on. I did encode them to be a lot smaller so they aren't too big.

"Focus" - Has artifacting across the image on fade to black/white, and artifacts on flashes (1:24~1:48)
"Problem" - Has artifacting on fast spinning moving line pattern (2:22~2:29)
"Side to Side" - Has lots of artifacting on wall even though the movement isn't too large and on fade to blacks (2:24~3:28)

Source clips (encoded)
https://drive.google.com/drive/folders/17NY9f2UKpV_JWPq5Z1f0Xa3o_T1L8fRU?usp=sharing

If the model can even just improve those type of things that would be amazing. These are the only easily noticeable artifacts I've run into.
Thanks for the work you put into making the best FI AI.  :)

**Comment by Tsubajashi** [comment]
Sorry for the necro, but is that issue fixed on v3? Just interested.

**Comment by ChristianUndCo** [comment]
I just tested model 3.1. There is nearly no improve with patterns. Some stuff gets even worse. But upscaling from "576 SD content" to 720p or 1080p helps sometimes a little bit.

**Comment by Felix3322** [comment]
This is an automated reply and does not represent the author's views.
Due to long inactivity and no new actionable information, this issue will be closed automatically for now.
If you want to reopen, please reply with $reopen and include updated reproduction details/logs.
If you need a human maintainer to step in, please @hzwer.
